DEVELOPER.md

```diff
@@ -21,7 +21,7 @@ This guide explains how to set up an environment to develop and test the Vertica
 # Software Setup
 Prior to developing, the following software needs to be installed manually.  There is other software that needed, but it is downloaded through make targets in the repo's bin directory.
 
-- [docker](https://docs.docker.com/get-docker/)
+- [docker](https://docs.docker.com/get-docker/) (version 19.03 or newer)
```

Comment:
Based on the description of the PR, should this be version 23.0?

Reply:
Yes It should. I put 19.03 because it is the 1st version supporting buildx but it is very old. Let's put 23.0.